sys/include/net/dhcpv6/client.h

```diff
@@ -204,42 +204,26 @@ void dhcpv6_client_conf_prefix(unsigned netif, const ipv6_addr_t *pfx,
                                unsigned pfx_len, uint32_t valid,
                                uint32_t pref);
 
-/**
- * @brief   Checks if the given network interface is configured
- *          to use DHCPv6 IA_NA
- *
- * @param[in] netif     Network interface to check.
- *
- * @return  true, if the network interface is set up for IA_NA.
- */
-bool dhcpv6_client_check_ia_na(unsigned netif);
-
 /**
  * @brief   Configures a address lease that is provided by the server.
  *
  * @param[in] netif     Network interface the address was for.
  * @param[in] addr      The assigned address.
- *
- * @return sizeof(ipv6_addr_t) on success.
- * @return <0 on error.
+ * @param[in] valid     Valid lifetime of the address.
+ * @param[in] pref      Preferred lifetime of the address.
  */
-int dhcpv6_client_add_addr(unsigned netif, ipv6_addr_t *addr);
+void dhcpv6_client_conf_addr(unsigned netif, const ipv6_addr_t *addr,
+                             uint32_t valid, uint32_t pref);
```

Comment:
This could be `static inline`

```suggestion
static inline void dhcpv6_client_conf_addr(unsigned netif, const ipv6_addr_t *addr,
                                           uint32_t valid, uint32_t pref)
{
    dhcpv6_client_conf_prefix(netif, addr, IPV6_ADDR_BIT_LEN, valid, pref);
}
```

Reply:
Nice! Refactored in 1075b9db7d5395ea5f4c8a1e379dfd826501031a :)